rocAL_pybind/examples/rocAL_training_example/README.md

```diff
@@ -0,0 +1,113 @@
+# ImageNet training in PyTorch
+
+This implements training of popular model architectures, such as ResNet, AlexNet, and VGG on the ImageNet dataset.
+This version has been modified to use rocAL. It assumes that the dataset is raw JPEGs from the ImageNet dataset. If offers CPU and GPU based pipeline for rocAL - use rocal-cpu switch to enable CPU one and use rocal-gpu switch to enable GPU one.
+
+To run use the following command
+```bash
+rm *.pth.tar # Remove older checkpoints saved in the folder if the example has been run before
+python3 main.py -a resnet50 --dist-url='tcp://127.0.0.1:4321' --dist-backend 'nccl' --multiprocessing-distributed --world-size 1 --rank 0 -j$(nproc) --batch-size 1024 --rocal-cpu --epochs 91 /media/imageNetCompleteDataset/
+```
+
+## Requirements
+
+- Install PyTorch for [ROCm](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/how-to/3rd-party/pytorch-install.html)
+- Install rocAL for running rocAL trainings
+- Download the ImageNet dataset from http://www.image-net.org/
+  - Then, move and extract the training and validation images to labeled subfolders, using [the following shell script](extract_ILSVRC.sh)
+
+## Training
```

Comment:
seems like all this is coming from pytorch repo. We shouldn't be replicating line by line. Please give links

Reply:
Removed requirements.txt and extract_ILSVRC.sh files and updated readme to point to pytorch examples repo